pkg/api/vlabs/types.go

```diff
@@ -226,32 +254,6 @@ type KeyVaultCertificate struct {
 	CertificateStore string `json:"certificateStore,omitempty"`
 }
 
-// OrchestratorType defines orchestrators supported by ACS
-type OrchestratorType string
```

Comment:
1. Unnecessary
2. I am going to add a `OrchestratoryTypeHint`. If i follow the tradition, it will be another custom type. Don't feel right.